src/zigzagboom1d.jl

```diff
@@ -49,7 +49,9 @@ function pdmp(∇ϕ, x, θ, T, c, Flow::ContinuousDynamics; adapt=false, factor=
             push!(Ξ, (t, x, θ))
         else
             t, x, θ = move_forward(τ, t, x, θ, Flow)
-            τref -= τ
+            if Flow isa Boomerang1d
+                τref = waiting_time_ref(Flow) #-= τ
```

Comment:
@SebaGraz Does this make sense to you?

Reply:
No. I think you should just rescale τref since is an independent Poisson clock, independent from the reflection clock.

Reply:
We could also think to move forward, instead of moving backwards in time (similarly to what we are doing with the `LocalZigZag`). Then you don't need to rescale `τref`